web_database_rollback/static/src/js/db_rollback.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2019 Cojocaru Aurelian Marcel PFA
+// @author Marcel Cojocaru <marcel.cojocari@gmail.com>
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+openerp.web_database_rollback = function (instance) {
+
+    instance.web_database_rollback.RollbackButtonsWidget = instance.web.Widget.extend({
+
+        template:'web_database_rollback.ButtonsWidget',
+
+        renderElement: function() {
+            var self = this;
+            this._super();
+            this.$el.show();
+            this.$el.find('.activate').on('click', function(ev) {
+                    self.$el.find('.activate').css("background-color", "green").css("color", "white");
+                    var func = '/web_database_rollback/activate';
+                    self.rpc(func, {}).done(function(res) {
```

Comment:
Is it worth having `func` declared separated? If so, `func` is a bit misleading, maybe `endpoint` would be more precise? (same below)